### DISCUSSION TITLE
Tracking future intakes

### DISCUSSION THREAD

**Discussion by felinerescue** [sheltermanager/asm3#1567] [General]
Hello!

Does anyone use ASM to track upcoming intakes? If so, I'm wondering how you go about it and any useful tips. I would really like us to start entering our upcoming animals ahead of time so that records could be started early and everyone can see what's upcoming. Right now we're using a spreadsheet and it works fine, but I much prefer to have as much info as I can in one area. 

I know that you can select a future intake date and that there's a setting for "Treat animals with a future intake date as part of the shelter inventory". Though when I checked this box and made a test animal, they would not show up in the shelter inventory. 

If you do things this way, do you have an animal location that's something like "future intake"?

Should these animals be marked as "non-shelter" until entry?

I appreciate any insight. Thank you!

**Comment by Tetzor** [comment]
Hi, animals with a future entry date should appear in shelterview  and in the shelter inventory report when the option "Treat animals with a future intake date as part of the shelter inventory" is enabled. However if you have just enabled the option, in order to see the change you will need to trigger the overnight batch early to recalculate animal locations. you can do this in Settings->Trigger Batch processes->Recalculate on-shelter animal locations.  

Jon

**Comment by felinerescue** [comment]
Thanks, Jon. I just tried this and the cat is still not showing up. The checkbox is checked and the animal is named Future Test Cat with an entry date of 11/20. Is there anything else I'm missing that needs to be done?



Sara Post (she/her)

Operations Director



Cell:   612-887-5665 (texts ok)​

felinerescue.org<http://felinerescue.org/> | facebook.com/felinerescueinc<http://facebook.com/felinerescueinc> | instagram.com/felinerescue<http://instagram.com/felinerescue>

[Feline-Rescue-Logo-Horiz-Color - E-SIg]

________________________________
